flask_table/table.py

```diff
@@ -151,7 +156,7 @@ def add_column(cls, name, col):
         return cls
 
 
-def create_table(name=str('_Table'), base=Table):
+def create_table(name=str('_Table'), base=Table, options={}):
```

Comment:
I think I would rather the default on this line was `None`, and then in the function body default it to be `{}` because of the python-mutable-default-arguments gotcha.